packages/commands/src/store/reducer.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * WordPress dependencies
+ */
+import { combineReducers } from '@wordpress/data';
+
+/**
+ * Reducer returning the registered commands
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ *
+ * @return {Object} Updated state.
+ */
+function commands( state = {}, action ) {
+	switch ( action.type ) {
+		case 'REGISTER_COMMAND':
+			return {
+				...state,
+				[ action.group ]: {
+					...state[ action.group ],
+					[ action.name ]: {
+						name: action.name,
+						label: action.label,
+						group: action.group,
+						callback: action.callback,
+					},
+				},
+			};
+		case 'UNREGISTER_COMMAND': {
+			const { [ action.name ]: _, ...remainingState } =
+				state?.[ action.group ];
+			return {
+				...state,
+				[ action.group ]: remainingState,
+			};
+		}
+	}
+
+	return state;
+}
+
+/**
+ * Reducer returning the command loaders
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ *
+ * @return {Object} Updated state.
+ */
+function commandLoaders( state = {}, action ) {
+	switch ( action.type ) {
+		case 'REGISTER_COMMAND_LOADER':
+			return {
+				...state,
+				[ action.group ]: {
+					...state[ action.group ],
+					[ action.name ]: {
+						name: action.name,
+						hook: action.hook,
+					},
+				},
+			};
+		case 'UNREGISTER_COMMAND_LOADER': {
+			const { [ action.page ]: _, ...remainingState } = state;
```

Comment:
Something is wrong here. `.page` isn't a property sent by the action creator. I believe this should be `.group`, and only the loader keyed under `.name` should be popped, like we do for the `commands` reducer.

Reply:
yeah, this is from a previous implementation, good catch.